doc/api/process.md

```diff
@@ -2309,6 +2309,227 @@ import { loadEnvFile } from 'node:process';
+
+```bash
+  EMPTY=
+```
+
+```cjs
+const assert = require('node:assert');
+const process = require('node:process');
+assert.strictEqual(process.env.EMPTY, '');
+```
+
+```mjs
+import assert from 'node:assert';
+import process from 'node:process';
+assert.strictEqual(process.env.EMPTY, '');
+```
+
+#### Quoted values
+
+* Single (`'`), double (`"`), and backtick (`` ` ``) quotes are recognized.
+  The content within the quotes is taken as is, including leading and
+  trailing spaces.
+* Quotes within a different quote type are preserved.
+
+```bash
+  FIRST_NAME=' John '
+  MIXED_QUOTES="Say 'Hello!'"
+  BACKTICK_IN_QUOTES="Using `backticks` in double quotes"
+```
+
+```cjs
+const assert = require('node:assert');
+const process = require('node:process');
+assert.strictEqual(process.env.FIRST_NAME, ' John ');
+assert.strictEqual(process.env.MIXED_QUOTES, "Say 'Hello!'");
+assert.strictEqual(process.env.BACKTICK_IN_QUOTES,
+                   'Using `backticks` in double quotes');
+```
+
+```mjs
+import assert from 'node:assert';
+import process from 'node:process';
+assert.strictEqual(process.env.FIRST_NAME, ' John ');
+assert.strictEqual(process.env.MIXED_QUOTES, "Say 'Hello!'");
+assert.strictEqual(process.env.BACKTICK_IN_QUOTES,
+                   'Using `backticks` in double quotes');
+```
+
+#### Multiline values
+
+* Values enclosed in quotes described above that span multiple
+  lines are considered valid and includes escaped new line characters.
+
+```bash
+ MULTI_DOUBLE_QUOTED="double
+ quoted"
+
+ MULTI_SINGLE_QUOTED='single
+ quoted'
+
+ MULTI_BACKTICKED=`this
+ "multiline"
+ value`
+```
+
+```cjs
+const assert = require('node:assert');
+const process = require('node:process');
+assert.strictEqual(process.env.MULTI_DOUBLE_QUOTED, 'double\nquoted');
+assert.strictEqual(process.env.MULTI_SINGLE_QUOTED, 'single\nquoted');
+assert.strictEqual(process.env.MULTI_BACKTICKED,
+                   'this\n"multiline"\nvalue');
+```
+
+```mjs
+import assert from 'node:assert';
+import process from 'node:process';
+assert.strictEqual(process.env.MULTI_DOUBLE_QUOTED, 'double\nquoted');
+assert.strictEqual(process.env.MULTI_SINGLE_QUOTED, 'single\nquoted');
+assert.strictEqual(process.env.MULTI_BACKTICKED,
+                   'this\n"multiline"\nvalue');
+```
+
+#### Escapes
+
+* Newlines in double-quoted values are expanded to newline characters.
+* Other escapes (e.g., `\n`) are treated as literal text in single-quoted
+  or unquoted values.
+
+```bash
+  EXPAND_NEWLINES="expand\nnew\nlines"
+  DONT_EXPAND_UNQUOTED=dontexpand\nnewlines
+  DONT_EXPAND_SQUOTED='dontexpand\nnewlines'
+```
+
+```cjs
+const assert = require('node:assert');
+const process = require('node:process');
+assert.strictEqual(process.env.EXPAND_NEWLINES, 'expand\nnew\nlines');
+assert.strictEqual(process.env.DONT_EXPAND_UNQUOTED,
+                   'dontexpand\\nnewlines');
+assert.strictEqual(process.env.DONT_EXPAND_SQUOTED,
+                   'dontexpand\\nnewlines');
```

Comment:
IMO, assertions shouldn't be used in the example, but rather just a comment explaining the item's value